rimage/draw.go

```diff
@@ -29,7 +29,7 @@ func Font() *truetype.Font {
 func DrawString(dc *gg.Context, text string, p image.Point, c color.Color, size float64) {
 	dc.SetFontFace(truetype.NewFace(Font(), &truetype.Options{Size: size}))
 	dc.SetColor(c)
-	dc.DrawString(text, float64(p.X), float64(p.Y))
+	dc.DrawStringWrapped(text, float64(p.X), float64(p.Y), 0, 0, float64(dc.Width()), 1, 0)
```

Comment:
@bhaney This stops the text from falling off the image, but is still a bit strange compared to standard image sizes. Here is a before the change:
![Screenshot 2023-04-04 at 3 17 37 PM](https://user-images.githubusercontent.com/121991867/230171257-1ace7421-db7a-4e14-b60a-2a9f5ea0a6e0.png)

And here is after the change:
![Screenshot 2023-04-05 at 2 23 39 PM](https://user-images.githubusercontent.com/121991867/230171299-3e36700d-31b8-4092-b486-53345306add3.png)

Thoughts?